sdk/eventhub/azure-eventhub/stress/stress_runner.py

```diff
@@ -7,6 +7,8 @@
 import configparser
 from subprocess import Popen
 from typing import cast
+from datetime import datetime
+import time
```

Comment:
unused imports?

Reply:
oops, good catch it was me trying to player with docker stuff adding some logic to deal with time.
removed